doc/26_Best_Practice/25_Where_To_Store_Sessions.md

```diff
@@ -0,0 +1,7 @@
+# Where To Store Sessions
+
+Pimcore uses the Symfony session component to handle sessions. By default, Symfony stores sessions in the filesystem.
+This is fine for most applications, but it can be a performance bottleneck for high-traffic sites.
+
+To avoid blocking requests, you can store sessions in a database or a cache.
+We strongly recommend using a cache like Redis to store sessions.
```

Comment:
Redis is not a “cache”. It is a key–value database ;)

Reply:
Also, maybe you should add a warning that it can lead to race-conditions if you do that (because it's non-blocking).